docs/modules/ROOT/pages/sql-on-viridian.adoc

```diff
@@ -197,61 +197,7 @@ ON companies.ticker = trades.ticker;
 
 ```
 
-== Step 4. Watermarking and Windowing
-
-Data aggregation is a common ETL function, but how do you do it on streaming data? The answer is to perform it over specific windows of time. The aggregations and computations are performed on all data included within each window. The output is then updated for each window. 
-
-[NOTE]
-====
-For a detailed description of watermarking and window types, refer to the https://docs.hazelcast.com/hazelcast/5.2/sql/querying-streams#windowing[SQL Stream Processing] topic in the Hazelcast documentation. 
-====
-
-. In order to ensure that data is included in the correct window, you have to first create a new view that orders the data based on one of the data fields (the watermark). In this tutorial, we'll use timestamp as the watermark. 
-+
-```sql
-CREATE OR REPLACE VIEW trades_ordered AS
-SELECT *
-  FROM TABLE(IMPOSE_ORDER(
-  TABLE trades,
-  DESCRIPTOR(trade_ts),
-  INTERVAL '0.5' SECONDS));
-```
-
-. For our first aggregation, we'll display the minimum and maximum price for each stock over a 5 second window. (Output will not appear until 5 seconds have elapsed.)
-+
-```sql
-SELECT 
-     window_start,
-     window_end, 
-     ticker, 
-     ROUND(MAX(price),2) AS high, 
-     ROUND(MIN(price),2) AS low
-FROM TABLE(TUMBLE(
-     TABLE trades_ordered,
-     DESCRIPTOR(trade_ts),
-     INTERVAL '5' SECONDS
-))
-GROUP BY 1,2,3
-;
-```
-This query will display the average price over a 5 second window, updating the result every second.
-+
-```sql
-SELECT 
-     window_start, 
-     window_end, 
-     ticker, 
-     ROUND(AVG(price),2) as average
-FROM TABLE(HOP(
-  TABLE trades_ordered,
-  DESCRIPTOR(trade_ts),
-  INTERVAL '5' SECONDS, INTERVAL '1' SECOND
-))
-GROUP BY 1,2,3;
-```
-
-
-== Step 5: Stream to Stream Joins
+== Step 4: Stream to Stream Joins
```

Comment:
@rebekah-lawrence Thanks for this PR!

I think I've explained "step 4" in a wrong way.
What I meant was, it'd better to move "Step 4" of "Step 1. Set up Streaming Data" after "Step 5" of "Step 1. Set up Streaming Data" section.
Since the code in "Step 3" of "Step 1. Set up Streaming Data" has the `CREATE OR REPLACE VIEW` it will complete in a few seconds, and the user shouldn't press Ctrl+C when that runs.

So, I think the ideal order in "Step 1. Set up Streaming Data" would be:
3. Use an SQL view to generate streaming data.
4. Verify that the data is streaming by issuing a query.
5. If you’re using the CLC SQL shell, press CTRL-C to return to the prompt.

Reply:
I was actually going to double-check whether that was what you meant when I looked at it yesterday, but got side-tracked and completely forgot today!

Ignore this one, I'll try again. Apologies.